app/src/main/java/org/squidfish/tuplenback/data/GameStatsMapper.kt

```diff
@@ -0,0 +1,29 @@
+package org.squidfish.tuplenback.data
+
+import java.lang.IllegalArgumentException
+import org.squidfish.tuplenback.data.room.GameStatsData
+import org.squidfish.tuplenback.models.GameStatsModel
+
+class GameStatsMapper {
+    companion object {
+        fun toData(stats: GameStatsModel): GameStatsData = GameStatsData(
+            gameEndTime = stats.gameEndTime,
+            gameType = stats.gameType ?: throw IllegalArgumentException("Game type is unset"),
+            gameModule = stats.gameModule ?: throw IllegalArgumentException("Game module is unset"),
```

Comment:
We have to talk about how we handle incorrect states. Do we use `try`/`catch` or a `Result` (though a custom one) class. The problem with `try`/`catch` is that the function signature does not indicate that an error is thrown, therefore you need to jump the source code to see if you need to use a `try`/`catch` block (sure, it can be documented, but still does not provide compile-time safety)